source/PcAux/R/exportedHelperFunctions.R

```diff
@@ -277,27 +277,31 @@ getImpData <- function(pcAuxData) pcAuxData$miDatasets
 calcTime <- function(pcAuxData, what) {
     time     <- pcAuxData$time[[what]]  
     eachStep <- diff(time)
-    
+    ##
     nPoints                      <- length(eachStep)
     eachStep[nPoints + 1]        <- as.vector(time[length(time)] - time["start"])
     names(eachStep)[nPoints + 1] <- "overall"
-
+    ##
     usrVars <- lapply(c("End", "usr"), function(x) grep(x, names(eachStep)))
-    
+    ##
     if(length(unlist(usrVars)) > 1) {
         eachStep["overall"] <- eachStep["overall"] - eachStep[usrVars[[1]]]
         timeSteps           <- eachStep[-usrVars[[2]]]
         timeSteps["usr"]    <- eachStep[usrVars[[1]]]
     }
     else timeSteps <- eachStep
-    
-    timeSteps
+    ##
+    return(timeSteps)
 }
 
 
 ## Write machine status to a text files 
 writeStatus <- function(pcAuxData, outName, what) {
```

Comment:
Also, we probably shouldn't return anything from the "writeStatus" function. We should have a separate "getStatus" function for that.

Reply:
Changed the return object to a character string that informs the user about the file writeStatus created.